core/src/test/java/io/questdb/test/cairo/view/ViewBootstrapTest.java

```diff
@@ -0,0 +1,746 @@
+                            "\"error\":\"views are disabled, set 'cairo.view.enabled=true' in the config to enable them\"," +
+                            "\"position\":7" +
+                            "}"
+            );
+
+            // cannot compile existing view via PG
+            assertExecRequest(
+                    httpClient,
+                    "compile view " + VIEW1,
+                    HTTP_BAD_REQUEST,
+                    "{" +
+                            "\"query\":\"compile view view1\"," +
+                            "\"error\":\"views are disabled, set 'cairo.view.enabled=true' in the config to enable them\"," +
+                            "\"position\":8" +
+                            "}"
+            );
+
+            // cannot drop existing view via PG
+            assertExecRequest(
+                    httpClient,
+                    "drop view " + VIEW1,
+                    HTTP_BAD_REQUEST,
+                    "{" +
+                            "\"query\":\"drop view view1\"," +
+                            "\"error\":\"views are disabled, set 'cairo.view.enabled=true' in the config to enable them\"," +
+                            "\"position\":5" +
+                            "}"
+            );
+        }
+    }
+
+    private static void assertExecRequest(
+            HttpClient httpClient,
+            String sql,
+            int expectedHttpStatusCode,
+            String expectedHttpResponse
+    ) {
+        final HttpClient.Request request = httpClient.newRequest("localhost", HTTP_PORT);
+        request.GET().url("/exec").query("query", sql);
+        assertHttpRequest(request, expectedHttpStatusCode, expectedHttpResponse);
+    }
+
+    private static void assertHttpRequest(
+            HttpClient.Request request,
+            int expectedHttpStatusCode,
+            String expectedHttpResponse,
+            String... expectedHeaders
+    ) {
+        try (HttpClient.ResponseHeaders responseHeaders = request.send()) {
+            responseHeaders.await();
+
+            assertEquals(String.valueOf(expectedHttpStatusCode), responseHeaders.getStatusCode());
+
+            for (int i = 0; i < expectedHeaders.length; i += 2) {
+                final Utf8Sequence value = responseHeaders.getHeader(new Utf8String(expectedHeaders[i]));
+                assertTrue(Utf8s.equals(new Utf8String(expectedHeaders[i + 1]), value));
+            }
+
+            final StringSink sink = tlSink.get();
+
+            Fragment fragment;
+            final Response chunkedResponse = responseHeaders.getResponse();
+            while ((fragment = chunkedResponse.recv()) != null) {
+                Utf8s.utf8ToUtf16(fragment.lo(), fragment.hi(), sink);
+            }
+
+            TestUtils.assertEquals(expectedHttpResponse, sink);
+            sink.clear();
+        }
+    }
+
+    private static void assertSqlFailureViaPG(String sql, String expectedErrorMessage) {
```

Comment:
another constant parameter value

Reply:
i think it is unlikely to happen but if this warning is still here when the PR is fully ready for review, will inline it